src/itype.h

```diff
@@ -441,7 +441,7 @@ struct islot_gunmod : common_ranged_data {
     int sight_dispersion = -1;
 
     /**
-     *  If set (non-zero) mod functions as sight when recoil above mod @ref sight_dispersion */
+     *  If set (non-negative) mod functions as sight when recoil above mod @ref sight_dispersion */
     int aim_cost = 0;
```

Comment:
The default is still 0 here.
Also, setting default to something other than 0, -1 or other "does not work" mandates commenting what does that value mean.